src/main/java/jenkinsci/plugins/influxdb/generators/AbstractPointGenerator.java

```diff
@@ -24,6 +26,7 @@ public AbstractPointGenerator(MeasurementRenderer projectNameRenderer) {
         Point.Builder builder = Point
                 .measurement(name)
                 .addField(PROJECT_NAME, renderedProjectName)
+                .addField(PROJECT_PATH, build.getParent().getRelativeNameFrom(Jenkins.getInstance()))
```

Comment:
@asimell I suggest that `PROJECT_PATH` is also added as tag. Otherwise it's not possible to filter for or group by it.

Reply:
An even better alternative could be to remove `PROJECT_PATH` (tag and field) completely and add a tag like `PARENT` which contains the parent folder like `folder1/subfolder5` of a job with path `folder1/subfolder5/job2`.

With this more normalized schema it would be possible to filter and group by the parent (which could be for example folders for different areas in Jenkins or the main folder/job of a multi-branch job where each branch / pull request / tag is a separate job).

What's your opinion on this @asimell and @dcoraboeuf?